### PR TITLE
Updated Error Messages for #1

### PR DESCRIPTION
**Pull request by haydenty**
Made the error messages more direct. Per issue #1  
Hope this helps! 
Happy coding!

### CODE REVIEW

**Comment by mbad0la** [comment]
@haydenty you should have commented on the issue first, as two other people were already working on this.

**Comment by cezarlz** [comment]
Hi @haydenty , thank you so much! Let me take a look on messages, and I will merge if ok.

Thanks! ;)

**Comment by cezarlz** [comment]
@haydenty can you update README too?

**Comment by haydenty** [comment]
@mbad0la sorry for not commenting, I'll try to do that next time. I thought for this type of issue multiple people could submit PR and @cezarlz could decide which one was the best. 

@mbad0la  I fixed the alpha.
@cezarlz I can take a peek at the README as well. What are you all thinking in regards to updating it, just check grammer and content?

**Comment by mbad0la** [comment]
@haydenty the the 'alpha' message is better now IMO. By the way, what you do you think of the commas after 'Please' in the messages. 

Do you think it is an improper punctuation use?

**Comment by haydenty** [comment]
@mbad0la I was thinking about the comma as well and wasn't quite sure. It seems like it could go both ways.

**Comment by cezarlz** [comment]
Guys, please, just review the messages, some messages has a dot on the end and some doesn't have. What do you think? All messages must contains a dot on the end?

Good work guys! I'm really happy :D

File: src/js/boss.messages.js [hunk 1]
```diff
@@ -12,26 +12,26 @@ const messages = {
...
 
   // Strings
-  exact: 'Please, this field needs to have {val} characters.',
-  extensions: 'Please, upload a file with some of these extensions: {val}.',
-  contains: 'Please, this field needs to have the value: {val}.',
-  minlength: 'Please, this field needs minimum {val} characters.',
-  maxlength: 'Please, this field needs maximum {val} characters.',
-  starts: 'Please, this field needs to start with "{val}".',
-  ends: 'Please, this field needs to end with "{val}".',
+  exact: 'Must contain the following characters: {val}',
```
Comment (haydenty) [inline src/js/boss.messages.js]: No dot here since the user might think it is a character

File: src/js/boss.messages.js [hunk 2]
```diff
@@ -12,26 +12,26 @@ const messages = {
...
+  ends: 'This field should end with "{val}".',
 
   // Booleans
   boolean: 'This field needs to be "true" or "false".',
 
   // Regex
-  email: 'Please, provide a valid email address.',
-  url: 'Please, provide a valid URL address with http:// or https://.',
-  https: 'Your URL must starts with https://',
+  email: 'Please, provide a valid email address. (ex: user@gmail.com)',
```
Comment (haydenty) [inline src/js/boss.messages.js]: I didn't want to add a dot to confuse the user whether or not the url needed a dot at the end (it shouldn't)

File: src/js/boss.messages.js [hunk 3]
```diff
@@ -12,26 +12,26 @@ const messages = {
...
+  url: 'Please, provide a valid URL address with http:// or https://',
```
Comment (haydenty) [inline src/js/boss.messages.js]: same as above reasoning

File: src/js/boss.messages.js [hunk 4]
```diff
@@ -12,26 +12,26 @@ const messages = {
...
+  https: 'Your URL must start with https://',
```
Comment (haydenty) [inline src/js/boss.messages.js]: same as above reasoning

File: src/js/boss.messages.js [hunk 5]
```diff
@@ -12,26 +12,26 @@ const messages = {
...
   credit_card: 'Please, enter a valid credit card number.',
-  ip_v4: 'Please, enter a valid IPV4 address.',
-  ip_v6: 'Please, enter a valid IPV6 address.',
-  alpha: 'Only alpha characters are allowed.',
-  alpha_numeric: 'Only alpha numeric characters are allowed.',
+  ip_v4: 'Please, enter a valid IPV4 address. (ex: 172.16.254.1)',
```
Comment (haydenty) [inline src/js/boss.messages.js]: same as above reasoning

File: src/js/boss.messages.js [hunk 6]
```diff
@@ -12,26 +12,26 @@ const messages = {
...
+  ip_v6: 'Please, enter a valid IPV6 address. (ex: 3ffe:1900:4545:3:200:f8ff:fe21:67cf)',
```
Comment (haydenty) [inline src/js/boss.messages.js]: same as above

File: src/js/boss.messages.js [hunk 7]
```diff
@@ -12,26 +12,26 @@ const messages = {
...
+  alpha: 'Only letters are allowed.',
+  alpha_numeric: 'No special characters allowed, just numbers and letters',
```
Comment (haydenty) [inline src/js/boss.messages.js]: should have a dot, I will add this

**Comment by cezarlz** [comment]
Great work @haydenty

I agree with you!

**Comment by mbad0la** [comment]
@haydenty I think it would be better to remove the commas as I can see from this [discussion](http://english.stackexchange.com/questions/389/in-what-cases-should-i-use-a-comma-after-please) at stackexchange.

@cezarlz merge this after @haydenty confirms whether he's fine with the current messages or he'd add another commit to remove the commas after "Please". Everything else looks fine to me. Good work :+1:

**Comment by cezarlz** [comment]
Nice discussion!

@haydenty, take a look and tell what do you think, please.

**Comment by haydenty** [comment]
@mbad0la  nice find! I changed them so this should be good to pull request @cezarlz

**Pull request merged**